stepconf/strings.go

```diff
@@ -42,7 +45,12 @@ func toString(config interface{}) string {
 
 	str := fmt.Sprint(colorstring.Bluef("%s:\n", strings.Title(t.Name())))
 	for i := 0; i < t.NumField(); i++ {
-		str += fmt.Sprintf("- %s: %s\n", t.Field(i).Name, valueString(v.Field(i)))
+		field := t.Field(i)
+		var key, _ = parseTag(field.Tag.Get("env"))
```

Comment:
This feels risky to me. We'd like to make 1000% sure that printing this stuff won't break the step. Maybe error handling with fallback to old name would be safer? 🤔 (Disclaimer I haven't actually dived deep in the code, if this remark doesn't make sense please just ignore.)

Reply:
Discussed offline, resolving